src/components/Dashboard.vue

```diff
@@ -60,11 +72,59 @@ export default {
   },
   data() {
     let user = UserService.getUser() || {};
+    var subtopics = new Object();
+    subtopics['math'] = ['Algebra', 'Geometry', 'Trigonometry', 'Precalculus', 'Calculus'];
+    subtopics['esl'] = ['ESL'];
+    subtopics['college'] = ['General help'];
     return {
       user: user,
-      name: user.firstname || 'student'
+      name: user.firstname || 'student',
+      popUpStyle: { },
+      showHelpPopUp: false,
+      pickedTopic: '',
+      pickedSubtopic: '',
+      subtopics: subtopics,
+      coverStyle: { }
     }
   },
+  methods: {
+    getHelp() {
+      this.popUpStyle = {
```

Comment:
Better to apply most of these in the <style> section and only bind styles that will change through component interaction. For instance, you can set `display: none` and then only bind `display: flex` in this method